homeassistant/components/tuya/strings.json

```diff
@@ -638,6 +638,12 @@
       "filter_reset": {
         "name": "Filter reset"
       },
+      "pump": {
+        "name": "Pump"
+      },
+      "light_spectrum_veg_herb": {
+        "name": "Light spectrum for Veg & Herbs"
```

Comment:
I think this name is very specific to your device. The specification this refer to as a just a "Light".

Reply:
Updated